app/models/hipstergram.rb

```diff
@@ -0,0 +1,12 @@
+class Hipstergram < ActiveRecord::Base
+
+  belongs_to :hipster
+  has_many :like_before_cools
+
+  has_attached_file :image, :styles => { :large => "500x500", :medium => "300x300>", :thumb => "100x100>" }, :default_url => "/images/:style/missing.png"
+
+
+  validates_attachment_content_type :image, :content_type => /\Aimage\/.*\Z/
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)<br>Style/RegexpLiteral: Use `%r` around regular expression. (https://github.com/bbatsov/ruby-style-guide#percent-r)